lib/interface.rb

```diff
@@ -0,0 +1,96 @@
+class Interface
+
+  attr_reader :menu_array, :order
+
+  def initialize(filepath)
+    @menu = Menu.new(filepath)
+    @order = Order.new
+    @order_array = []
+    @menu.build_menu_display
```

Comment:
Could this method be called by Menu.initialize to increase Menu's encapsulation?